tests/integration/test_script_runner.py

```diff
@@ -357,6 +357,21 @@
         user_flow_script="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_sqlite_data.py",
         data_context_dir="tests/integration/fixtures/no_datasources/great_expectations",
     ),
+    IntegrationTestFixture(
+        name="how_to_instantiate_a_specific_filesystem_data_context",
+        user_flow_script="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_instantiate_a_specific_filesystem_data_context.py",
+        data_context_dir="tests/integration/fixtures/no_datasources/great_expectations",
```

Comment:
This test is also added in #7985. Could you have a closer look?

Reply:
@Shinnnyshinshin Indeed -- the second remark above mentions that reviewing will be easier if #7985 is merged first.  Thanks!